extensions/components/com_redcore/site/tables/translation.php

```diff
@@ -0,0 +1,146 @@
+<?php
```

Comment:
Really? Or is that based on using the admin model as you suggested doing instead?